### ISSUE TITLE
Writing while moving the player ball

### ISSUE THREAD

**Issue by haxballer** [haxball/haxball-issues#83]
Hey, while playing I noticed some difference between the new haxball version and the old one. Well, while moving with my ball, when I want to write something fast by tapping on the TAB, what I did on the flash version, the ball suddenly stops, so there is no way to write while moving, but on the old version was possible, it is simple gg type, wp etc, just that here when click enter although i keep the arrow, my ball stops. What do you think about it? Is it possible to modify this?

**Comment by Roll432** [comment]
do you mean that your player stop moving after pressing the enter key after writing message, and you need to press the arrow key again to continue moving? yea happens to me too. please fix that

**Comment by haxballer** [comment]
Yes, that's what I mean, but I don't know if it's a bug, or if it was supposed to be on this version of Haxball. I'm used to writing something while moving the ball without stopping. I don't know Basro's position on this, so I decided to mention it.
